runtime/include/chpl-comm.h

```diff
@@ -310,24 +316,27 @@ void chpl_gen_comm_wide_string_get(void *addr, c_nodeid_t node, void *raddr,
                                    int ln, c_string fn);
 
 //
-// remote fork should launch a thread on locale that runs function f
-// passing it arg where the size of arg is stored in arg_size
+// Launch a task on a remote locale to run function f, passing it
```

Comment:
I realize this flaw predates your work here, but this is a good opportunity to correct something.  We shouldn't say that chpl_comm_execute_on() "launch\[es\] a task on a remote locale". We should say something like it "runs function f(arg) on a remote locale". Whether a task is used to achieve this or not is up to the implementation and may not be the case.  In other words, whether a task is used or not isn't part of the interface definition, so we shouldn't say that here.